maps/site53/job/jobs/medical.dm

```diff
@@ -204,7 +209,7 @@
 	spawn_positions = 4
 	ideal_character_age = 40
 	economic_power = 5
-	//duties = "<big><b>As the EMT it is your job to man the medical post near the Class D cell block, and treat any injuries there of the guards or Class D's. You only have limited supplies, so it's best to make them count.</b></big>"
+	//duties = "<big><b>As the EMT it is your job to save as many people as possible and go into dangerous area and terrain to save wounded or dying personnel, but do be aware, your main job is to stabilize, so toss people to the doctors too.</b></big>"
```

Comment:
```suggestion
	//duties = "<big><b>As the EMT it is your job to respond to situations outside of the medical area. You should be the first medical staff to respond to an incident, and you should focus on stabilizing the wounded so that better prepared doctors can treat them. You have limited supplies, so it may be necessary to triage patients.</b></big>"
```
Run-on sentence. Also, this is closer to what an EMT should do.

Reply:
This is a comment -- who cares?